crates/bevy_render/src/lib.rs

```diff
@@ -1,3 +1,6 @@
+#[cfg(target_pointer_width = "16")]
+compile_error!("bevy_render cannot safely compile for a 16-bit platform.");
```

Comment:
Unlike bevy_ecs the lossy conversion here is not a matter of memory safety, so I think the message should be different.

Since bevy_render depends on bevy_ecs so the compile_error is also a bit redundant, but that's fine

Reply:
```suggestion
compile_error!("bevy_render cannot compile for a 16-bit platform.");
```

Meant this as extra redundancy in case we ever change that in `bevy_ecs` or if a user patches that out.